src/components/PhotoPage.js

```diff
@@ -21,17 +21,22 @@ import PageWrapper from './PageWrapper';
 import LinearProgress from '@material-ui/core/LinearProgress';
 import Link from '@material-ui/core/Link';
 
+let _errors = [];
+const _fields = [];
+Object.values(config.PHOTO_FIELDS).forEach(field => field.componentType === 'PhotoPageFieldText' && _fields.push(''));
```

Comment:
The problem with map is that it will create extra fields if field.componentType !== 'PhotoPageFieldText'.
(1)[""] 
(2) ["", false]
The false in this occasion is unnecessary